static/app/views/issueList/queries/useFetchGroupSearchViews.tsx

```diff
@@ -6,22 +6,17 @@ type FetchGroupSearchViewsParameters = {
   orgSlug: string;
 };
 
-type FetchGroupSearchViewsResponse = GroupSearchView[];
-
 export const makeFetchGroupSearchViewsKey = ({
   orgSlug,
 }: FetchGroupSearchViewsParameters) =>
   [`/organizations/${orgSlug}/group-search-views/`] as const;
 
 export const useFetchGroupSearchViews = (
   {orgSlug}: FetchGroupSearchViewsParameters,
-  options: Partial<UseApiQueryOptions<FetchGroupSearchViewsResponse>> = {}
+  options: Partial<UseApiQueryOptions<GroupSearchView[]>> = {}
 ) => {
-  return useApiQuery<FetchGroupSearchViewsResponse>(
-    makeFetchGroupSearchViewsKey({orgSlug}),
-    {
-      staleTime: Infinity,
-      ...options,
-    }
-  );
+  return useApiQuery<GroupSearchView[]>(makeFetchGroupSearchViewsKey({orgSlug}), {
```

Comment:
I left the useApiQuery cache type as just GroupSearchView[] instead of Required<GroupSearchView[]>, even though that cache should always have an id. Feels cleaner to make it the base interface, especially since they're compatible.

Reply:
Hmm I think that I would rather be more explicit and define two separate types. `GroupSearchView` should have a required ID and perfectly match what is returned from this endpoint. The update endpoint is the only thing that doesn't require an ID, so I'd make a new type in that file like:

```
interface UpdateGroupSearchViewPayload extends Omit<GroupSearchView, 'id'> {
  id?: string;
}
```